includes/admin/class-wc-payments-admin-sections-overwrite.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Overwrites the default payment settings sections in WooCommerce
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Payments_Admin Class.
+ */
+class WC_Payments_Admin_Sections_Overwrite {
+	/**
+	 * WC_Payments_Admin_Sections_Overwrite constructor.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_settings_page_init', [ $this, 'page_init' ] );
+		add_filter( 'woocommerce_get_sections_checkout', [ $this, 'add_checkout_sections' ] );
+	}
+
+	/**
+	 * Checks the current page in wp-admin to ensure we're looking at the payment gateways settings page.
+	 *
+	 * @return bool
+	 */
+	private function is_current_page_payment_methods() {
```

Comment:
This is not exactly what this method is checking - the check is now more of "was no specific section specified in the URL?". If a section is specified (whether _WooCommerce Payments_ or _All payment methods_), the method will return `false`.

Reply:
Ah, I see what you mean, thanks for pointing it out! Changed 👍